<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 20);</string>
  </property>
  <widget class="QLabel" name="balance">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(250, 126, 58);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(251, 155, 40);
font: 24pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(0, 0, 20);
border: 1px solid rgb(0, 0, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-bottom: 2px solid orange;</string>
   </property>
   <property name="text">
    <string>â‚¿ Your Wallet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-bottom: 2px solid green;</string>
   </property>
   <property name="text">
    <string>in $</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_usd">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(250, 126, 58);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: green;
font: 24pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(0, 0, 20);
border: 1px solid rgb(0, 0, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>General information on your wallet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;.AppleSystemUIFont&quot;;
border-bottom: 2px solid rgb(113, 183, 255);</string>
   </property>
   <property name="text">
    <string>Total received</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;.AppleSystemUIFont&quot;;
border-bottom: 2px solid rgb(113, 183, 255);</string>
   </property>
   <property name="text">
    <string>Total send</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalrec">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(250, 126, 58);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(251, 155, 40);
font: 24pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(0, 0, 20);
border: 1px solid rgb(0, 0, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalsend">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(250, 126, 58);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(251, 155, 40);
font: 24pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(0, 0, 20);
border: 1px solid rgb(0, 0, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="receive_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid rgb(92, 212, 156);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>RECEIVE</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(92, 212, 156);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Corbel&quot;;
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>Blockatom wallet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="walletbutton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(92, 212, 156);
background-color: rgb(23, 63, 99);
border-radius: 7px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/wallet.png</normaloff>images/wallet.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sendd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(92, 212, 156);
background-color: rgb(23, 63, 99);
border-radius: 7px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/rising.png</normaloff>images/rising.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="receive_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(92, 212, 156);
background-color: rgb(23, 63, 99);
border-radius: 7px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/receive.png</normaloff>images/receive.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(92, 212, 156);
background-color: rgb(23, 63, 99);
border-radius: 7px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/setting.png</normaloff>images/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(92, 212, 156);
background-color: rgb(23, 63, 99);
border-radius: 7px;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/user.png</normaloff>images/user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
